src/medCore/gui/database/medDatabaseView.cpp

```diff
@@ -559,13 +559,23 @@ void medDatabaseView::onEditRequested(void)
         QList<QVariant> attributes = item->attributes();
         QList<QVariant> values = item->values();
         QList<QString> labels;
-        
+
+        // The last attribute is ThumbnailPath. Users are not allowed to change it.
+        attributes.removeLast();
+        values.removeLast();
```

Comment:
In fact, thumbnailpath have been used some month/year ago (by me) in this list. I do not remember why it was needed, but it's a fixed list, and the last parameter is and has to be ThumbnailPath. As you want if you still want a search in this list